porting/finalize/UBports-official-support/port-support-requirements.rst

```diff
@@ -0,0 +1,102 @@
+.. _PortSupportRequirements:
+
+Port support requirements
+==================================
+
+This document enumerates the requirements that must be met in order for devices to be deemed ship ready for UBports releases.
+To be considered ready, port maintainers MUST meet the requirements presented in this document, including any documents incorporated via reference.
+
+The use of “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” is per the IETF standard defined in :RFC:`2119`.
+
+-----
+
+Attaining exceptions
+********************
+
+Process
+-------
+
+* Device or software deviations from our core product (as described below) MUST be approved by the devrel team.
+* Device or software exceptions SHOULD be made via change request to this repository.
+* All device or software exceptions that are granted MUST be documented below at the `Ubuntu Touch device page`_ for all affected devices.
+
+Hardware support
+****************
+
+Audio
+-----
+
+* All devices MUST support audio playback for media content.
+* Phones MUST support in-call audio.
+* Phones MUST support speaker audio.
+* Tablet devices capable of in-call audio/speaker audio MUST support in-call/speaker audio.
+
+RIL
```

Comment:
please spell out RIL. I have no clue what this is. If you think it's clear once it is spelled out, great, otherwise consider explaining/linking an explanation

Reply:
I have spelled it out, however i doubt explaining it further helps, unless we got a specific RIL section in our documentation already, which we want to link.

Otherwise the requirements get too verbose.

Reply:
agree. no point in elaborating more *here*. thanks for spelling it out. it at least gives a little more context for an internet search